src/main/java/io/github/mzmine/datamodel/features/types/graphicalnodes/FeatureShapeIonMobilityRetentionTimeHeatMapChart.java

```diff
@@ -81,23 +77,8 @@ public FeatureShapeIonMobilityRetentionTimeHeatMapChart(@NotNull ModularFeature
         .setRange(RangeUtils.guavaToJFree(RangeUtils.getPositiveRange(dataset.getDomainValueRange(), 0.001d)), false, true);
     chart.getXYPlot().getRangeAxis()
         .setRange(RangeUtils.guavaToJFree(RangeUtils.getPositiveRange(dataset.getRangeValueRange(), 0.0001d)), false, true);
-    BufferedImage img = chart.getChart()
-        .createBufferedImage(GraphicalColumType.LARGE_GRAPHICAL_CELL_WIDTH,
-            GraphicalColumType.DEFAULT_GRAPHICAL_CELL_HEIGHT);
 
-    ImageView view = new ImageView(SwingFXUtils.toFXImage(img, null));
-    view.setOnMouseClicked(e -> MZmineCore.runLater(() -> {
-      // change buffered image to buffered chart on mouse click
-      getChildren().remove(view);
-      getChildren().add(chart);
-      e.consume();
-    }));
-
-    Platform.runLater(() -> getChildren().add(view));
-
-//    chart.addDatasetsChangedListener(
-//        (e) -> Platform.runLater(() -> chart.getXYPlot().getRangeAxis().setAutoRange(true)));
-//    getChildren().add(chart);
-//    Platform.runLater(() -> chart.setDataset(dataset));
+    setChartCreateImage(chart, GraphicalColumType.LARGE_GRAPHICAL_CELL_WIDTH,
+        GraphicalColumType.DEFAULT_IMAGE_CELL_HEIGHT);
```

Comment:
i dont think this needs to be image height

Reply:
true not sure why I missed this